tests/all/utils/fixture.rs

```diff
@@ -74,6 +74,44 @@ impl Fixture {
         )
     }
 
+    /// Add `WTFPL LICENSE` file to the fixture.
+    pub fn wtfpl_license(&self) -> &Self {
+        self.file(
+            "LICENSE-WTFPL",
+            r#"
+                DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
+                    Version 2, December 2004
+
+                Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>
```

Comment:
can we just remove the text of this license? i know it's not something you personally wrote and is the text of the license but it's something i just don't want in this code base. feel free to replace it with a neutral sentence or just a copyright line.

Reply:
Yeah no problem, I was on the fence about this to begin with I'll go ahead and reword it!

Reply:
thank you!

Reply:
I changed all the lines containing this wording to use`WHATEVER` instead!